docker/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+database:
+  image: "quay.io/galexrt/zulip-postgresql-tsearchextras:latest"
+  environment:
+    DB_NAME: zulip
+    DB_USER: zulip
+    DB_PASS: zulip
+  volumes:
+    - "/opt/docker/zulip/postgresql/data:/var/lib/postgresql:rw"
+memcached:
+  image: "quay.io/sameersbn/memcached:latest"
+  restart: always
+rabbitmq:
+  image: "docker.io/rabbitmq:3.5.5"
```

Comment:
I had to remove the `docker.io` part of this image to get it to pull.  Just using `image: "rabbitmq:3.5.5"` will pull from the official docker hub image.